susemanager-utils/susemanager-sls/salt/images/kiwi-image-build.sls

```diff
@@ -91,6 +93,13 @@ mgr_buildimage_kiwi_bundle:
 {%- endfor -%}
 {%- endmacro %}
 
+# old Kiwi can't change cache location, so we have to clear cache before each build
+mgr_kiwi_clear_cache:
+  file.directory:
+    - name: /var/cache/kiwi/
```

Comment:
Why not `/var/cache/kiwi/zypper` as it was before?

Reply:
There is also /var/cache/kiwi/packages which would be filled with packages and never deleted. Let's make sure we use correct packages freshly downloaded from SUMA.

Reply:
Thanks for the explanation! Looks good.